manifests/operator.yaml

```diff
@@ -584,7 +584,7 @@ spec:
         - --web.listen-address=:19090
         - --web.enable-lifecycle
         - --web.route-prefix=/
-        - --export.user-agent=prometheus/2.35.0-gmp.2 (mode:kubectl)
```

Comment:
We chatted offline and it seems that `prometheus/2.35.0-gmp.2` is captured in another place. Can you point it out and resolve this comment here?

Reply:
Yep! It's input in the Prometheus fork here: https://github.com/GoogleCloudPlatform/prometheus/blob/v2.35.0-gmp.2/cmd/prometheus/main.go#L396